src/test/java/hometaskstream/PersonTest.java

```diff
@@ -0,0 +1,30 @@
+package hometaskstream;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class PersonTest {
+
+    @Test
+    void getPersonFirstNames() {
+        List<Person> personList = new ArrayList<>();
+        {
```

Comment:
Please remove these brackets and reformat code for this class after that

Reply:
thanks, done